src/main/java/core/basesyntax/services/imp/ParserServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.services.imp;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.services.ParserService;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ParserServiceImpl implements ParserService {
+    private static final int USLESS_LINE = 0;
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_NAME_INDEX = 1;
+    private static final int AMOUNT_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> parse(List<String> linesFromInputFile) {
```

Comment:
```suggestion
    public List<FruitTransaction> parse(List<String> lines) {
```
Always keep in mind abstraction. Is it important for THIS method where these lines came from? Tomorrow it will be the internet.